.changeset/nervous-shirts-know.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/account": patch
+"fuels": patch
+---
+
+chore: deprecate `FUEL_NETWORK_URL` and `LOCAL_NETWORK_URL`
```

Comment:
I'd say to hold this until #2915 goes to `master`.

Then, merge this against `master` as well, to keep history separate.

Reply:
* #2811

Yes - I agree.

Reply:
@petertonysmith94 Could we also release this independently before #2811?

Reply:
@arboleya I have been looking at this - it can be, I'll just be pulling over files from #2811 as we use `FUEL_NETWORK_URL` all over our tests.

Reply:
Got it, never mind then. I was just exploring possibilities.